config/autoscaler-hpa.yaml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: autoscaler-hpa
+  namespace: knative-serving
+  labels:
+    serving.knative.dev/release: devel
```

Comment:
Can we label this similar to the networking controllers, so that we can include/exclude it more easily?

I wonder if we should split it out in the release as well...

Reply:
We can, got a proposal for a label here? The other manifests have all been somewhat mutually exclusive (you only have one ingress-provider for example) so I‘m not quite sure which one to choose here.

Splitting out wise I guess we can do at a later step possibly as well. Keeping it in the overall YAML is not a change for now as it was there before.

Reply:
How about: `autoscaling.knative.dev/autoscaler-provider: hpa`?  Fits the pattern